pythonx/netranger/fs.py

```diff
@@ -178,11 +178,11 @@ def on_stderr(job_id, err_msg):
             pickle.dump(arguments, f)
 
         if sudo:
-            Vim.AsyncRun('sudo {} {} {}'.format(self.ServerCmd, cmd, fname),
+            Vim.AsyncRun('sudo {} {} {}'.format(self.ServerCmd, cmd, fname).replace('\\','\\\\'),
```

Comment:
It's better to put the replace in [Vim.AsyncRun](https://github.com/ipod825/vim-netranger/blob/50dbae4987a76311bd772d6d0c6b256b49adecc7/pythonx/netranger/Vim.py#L162)